dev/com.ibm.ws.com.unboundid/build.gradle

```diff
@@ -8,10 +8,17 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-
  jar.doFirst {
```

Comment:
Thought we could get rid of this jar.doFirst since we do the copy in the Include-Resources in the bnd.bnd.